backend/views/syson-diagram-general-view/src/main/java/org/eclipse/syson/diagram/general/view/nodes/DefinitionNodeDescriptionProvider.java

```diff
@@ -73,14 +73,11 @@ public void link(DiagramDescription diagramDescription, IViewDiagramElementFinde
 
         var allTargetNodeDescriptions = new ArrayList<NodeDescription>();
 
-        GeneralViewDiagramDescriptionProvider.DEFINITIONS.forEach(definition -> {
-            var optNodeDescription = cache.getNodeDescription(GVDescriptionNameGenerator.getNodeName(definition));
-            allTargetNodeDescriptions.add(optNodeDescription.get());
-        });
-
-        GeneralViewDiagramDescriptionProvider.USAGES.forEach(usage -> {
-            var optNodeDescription = cache.getNodeDescription(GVDescriptionNameGenerator.getNodeName(usage));
-            allTargetNodeDescriptions.add(optNodeDescription.get());
+        GeneralViewDiagramDescriptionProvider.TOOL_SECTIONS.forEach((sectionName, elements) -> {
```

Comment:
Why did you use TOOL_SECTIONS here? It seems strange to use the TOOL_SECTIONS to compute the node descriptions list that will be used by this Definition Node Palette.

Reply:
You are right I reverted this change